.github/workflows/test.yml

```diff
@@ -48,6 +48,9 @@ jobs:
       - name: Install npm dependencies
         run: npm ci
 
+      - name: Compile typescript
+        run: tsc
```

Comment:
Might make more sense to have this in an npm script and call it before publish or something, to make sure it's always the latest.